backend/tests/services/knowledge/code_wiki/test_source.py

```
@@ -0,0 +1,181 @@
+            return_value={"type": "github", "token": "t0ken"},
+        ),
+        patch(
+            "app.services.knowledge.code_wiki_source._provider_for",
+            return_value=provider,
+        ),
+    ):
+        result = assert_user_can_read_source(MagicMock(), 1, GITHUB_SOURCE)
+
+    assert result["has_access"] is True
+    provider.check_user_project_access.assert_called_once_with(
+        token="t0ken", git_domain="github.com", repo_name="wecode-ai/Wegent"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to argument: "token"

(S106)